src/VisualStudio/Core/Def/DocumentOutline/VirtualizingTreeView.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Windows.Automation.Peers;
+using System.Windows.Controls;
+
+namespace Microsoft.VisualStudio.LanguageServices.DocumentOutline;
+
+internal class VirtualizingTreeView : TreeView
```

Comment:
Consider just adding a comment to the file for the bug it's working around, rather than having to dig to see where it is.

Reply:
➡️ Done